### ISSUE TITLE
Custom `ctag`

### ISSUE THREAD

**Issue by WoosterInitiative** [AdguardTeam/AdGuardHome#5143]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

#### What problem are you trying to solve?
Client "groups" that are not well represented by built-in tags.

#### Proposed solution
Two options:

1. Add a few more options, specifically an "IoT" or "Smart Home" device type.
2. Allow for custom tags, perhaps separately from ctags to prevent pollution of their intended purpose.

#### Additional information
I've got a number of "smart home" or "IoT" devices (Nest Thermostat, Wyze Cameras, etc.) that are connected via WiFi and have services that they like to connect to. Of course, I can setup individual rules for each device, but there may be specific things that I want **all** IoT to avoid, or maybe all Wyze cameras to avoid (device_manufacturer ctag?).

Basically, a way to group clients in ways that the current inflexible system does not allow.

**Comment by g33k247** [comment]
Thank you, [WoosterInitiative](https://github.com/WoosterInitiative). I find the ctag attribute limiting as well. I like "device_iot", "device_smarthome", and would add "device_server"!

**Comment by VsevolodNaumov** [comment]
Duplicate of #3086
